packages/pds/src/db/migrations/20240117T001106576Z-phone-verification.ts

```diff
@@ -0,0 +1,18 @@
+import { Kysely } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .createTable('phone_verification')
+    .addColumn('did', 'varchar', (col) => col.primaryKey())
+    .addColumn('phoneNumber', 'varchar', (col) => col.notNull())
+    .execute()
+  await db.schema
+    .createIndex('phone_verification_number_idx')
+    .on('phone_verification')
+    .column('phoneNumber')
```

Comment:
We'll want to ensure the numbers stored here are normalized.

Reply:
ah right that's a good point 🤔

Reply:
okay actually took a different track on this & decided to just enforce the format that twilio wants (https://www.twilio.com/docs/glossary/what-e164#regex-matching-for-e164)

the api forces it pretty close, but it still allows some slight variations (like dashes between numbers)